ext/Enzyme012Ext.jl

```diff
@@ -0,0 +1,342 @@
+# https://github.com/EnzymeAD/Enzyme.jl/issues/1516
```

Comment:
Can we name these files after the EnzymeCore version?

Reply:
done